AElf.Kernel.Consensus/Application/IConsensusScheduler.cs

```diff
@@ -0,0 +1,10 @@
+using AElf.Common;
+
+namespace AElf.Kernel.Consensus.Application
+{
+    public interface IConsensusScheduler
+    {
+        void Launch(int countingMilliseconds, int chainId, Hash preBlockHash, ulong preBlockHeight);
```

Comment:
implement IChainRelatedComponent in next pr

Reply:
OK